rc/base/screen.kak

```diff
@@ -0,0 +1,48 @@
+# http://gnu.org/software/screen/
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+ 
+hook global KakBegin .* %{
+    %sh{
+        [ -z "$STY"] && exit
+        echo "
+            alias global focus screen-focus
+            alias global new screen-new-vertical
+        "
+    }
+}
+
+
+def screen-new-vertical -params .. -command-completion -docstring "Create a new vertical region" %{
+     %sh{
+       cmd="screen -X eval 'split -h' 'focus down' 'screen kak -c ${kak_session} -e \"$@\"'"
```

Comment:
* Why not use `eval` directly there?
* `$@` doesn't make sense here, you need to use `$*`
* `${kak_session}` needs to be enclosed in quotes